### PR TITLE
feat(onesync): update data node CVehicleDamageStatus

### PR DESCRIPTION
**Pull request by ahcenezdh**
Clean-up of CVehicleDamageStatusDataNode completed; most of the data were already present.

### Checklist

- [x] Code compiles and has been tested successfully.
- [x] Code explains itself well and/or is documented.
- [x] My commit message explains what the changes do and what they are for.
- [x] No extra compilation warnings are added by these changes.

### CODE REVIEW

File: code/components/citizen-server-impl/include/state/SyncTrees_Five.h [hunk 1]
```diff
@@ -857,44 +857,26 @@ struct CVehicleDamageStatusDataNode
...
-		}
-
 		data.damagedByBullets = state.buffer.ReadBit();
 
 		if (data.damagedByBullets)
 		{
 			for (int i = 0; i < 6; i++)
 			{
-				uint8_t bulletsCount = state.buffer.Read<uint8_t>(8);
+				data.bulletsCount = state.buffer.Read<uint8_t>(8);
```
Comment (ahcenezdh) [inline code/components/citizen-server-impl/include/state/SyncTrees_Five.h:866]: Just need to fix that and should be good

**Comment by Korioz** [comment]
Most of your changes are incorrect... Also using data from *you know what* to code something that's gonna be read by R* devs doesn't help the situation.

**Comment by ahcenezdh** [comment]
Ohh.. at least i tried lol

**Comment by Disquse** [comment]
The "has been tested successfully" point is not just for fun here :P

**Pull request closed without merging**